plasma_framework/contracts/src/exits/payment/PaymentStandardExitable.sol

```diff
@@ -257,18 +257,17 @@ contract PaymentStandardExitable is
     }
 
     function enqueueStandardExit(StartStandardExitData memory data) private {
-        uint256 exitableAt = exitableTimestampCalculator.calculate(
+        uint32 exitableAt = exitableTimestampCalculator.calculate(
```

Comment:
Why do you use only 32 bits? Don't  we have space for additional 32?

Reply:
Originally was from this comment: https://github.com/omisego/plasma-contracts/blob/master/contracts/RootChain.sol#L1104

And it seems like a uint32 would be able to serve us until Sunday, February 7, 2106, which I think is long enough for now.

However, as you mentioned and I checked, the standard timestamp is using signed integer instead of unsigned then would lead to [Year 2038 problem](https://en.wikipedia.org/wiki/Year_2038_problem). Given that, I think it's better we bump it to 64 instead of 32 though if we keep it "unsigned" we are probably quite fine.